pages/ksphere/kommander/latest/operations/cloud-providers/index.md

```diff
@@ -144,7 +125,49 @@ Below is the minimal IAM policy required:
 }
 ```
 
-#### Fill out the rest of the form
+### Role-based Authentication Method
+
+<p class="message--note"><strong>NOTE: </strong>The Role authentication method can only be used if your management cluster is running in AWS.</p>
+
+To utilize this option you going to need to attach the role to the instances your Kommander cluster is running. Unless you created the Kommander cluster [using the Instance Profile](/ksphere/konvoy/latest/install/install-aws/advanced-provisioning/#iam-instance-profiles) already.
```

Comment:
This is not right.
Another role created does not need to be attached to the instances.
But at least you should have, next to other policies the Kommander cluster needs anyway, the following policy attached to the same role the cluster has attached at the moment: 

```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "AssumeRoleKommander",
            "Effect": "Allow",
            "Action": "sts:AssumeRole",
            "Resource": "arn:aws:iam::YOURACCOUNTRESTRICTION:role/THEROLEYOUCREATED"
        }
    ]
}
```

Reply:
With this one I need a little help. I don't quite understand how to change the paragraph :)

Reply:
please remove the backslashes I used for escaping the suggestions content 😬 

```suggestion
To utilize this option you need to attach the following policy to the role which is attached to your Kommander cluster.
\`\`\`json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "AssumeRoleKommander",
            "Effect": "Allow",
            "Action": "sts:AssumeRole",
            "Resource": "arn:aws:iam::YOURACCOUNTRESTRICTION:role/THEROLEYOUCREATED"
        }
    ]
}
\`\`\`
Instead of doing this manually you can add the following file `iam-stsAssumeKommander.tf` into the `extras/provisioner/` directory next to your `cluster.yaml`
\`\`\`terraform
# Attaching sts:AssumeRole to the default node role from konvoy
resource "aws_iam_role_policy" "agent_policy_assumerole_kommander" {
  count = "${var.create_iam_instance_profile ? 1 : 0}"
  name  = "AssumeRoleKommander"
  role  = "${aws_iam_role.node_role.id}"

  policy = <<EOF
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "AssumeRoleKommander",
            "Effect": "Allow",
            "Action": "sts:AssumeRole",
            "Resource": "arn:aws:iam::YOURACCOUNTRESTRICTION:role/THEROLEYOUCREATED"
        }
    ]
}
EOF
}
\`\`\`
```

Reply:
fixed thank you!